packages/instui-config/scripts/generate-package-list-command.mjs

```diff
@@ -24,14 +24,15 @@
  * SOFTWARE.
  */
 
-const path = require('path')
-const semver = require('semver')
-const { getPackageJSON } = require('@instructure/pkg-utils')
-const { error } = require('@instructure/command-utils')
-const generatePackageList = require('./generate-package-list')
+import path from 'path'
```

Comment:
this package did not needed to be converted to ES6, I just did it while checking some stuff